docs/guide/custom_policy.rst

```diff
@@ -5,7 +5,33 @@ Custom Policy Network
 
 Stable baselines provides default policy networks (see :ref:`Policies <policies>` ) for images (CNNPolicies)
 and other type of input features (MlpPolicies).
-However, you can also easily define a custom architecture for the policy (or value) network:
+
+One way of customising the policy network architecture is to pass argument using ``policy_kwargs`` parameter:
+
+.. code-block:: python
+
+  import gym
+  import tensorflow as tf
+
+  from stable_baselines import PPO2
+
+  # Custom MLP policy of two layers of size 32 each with tanh activation function
+  policy_kwargs = dict(act_fun=tf.nn.tanh, net_arch=[32, 32])
+  # Create the agent
+  model = PPO2("MlpPolicy", "CartPole-v1", policy_kwargs=policy_kwargs, verbose=1)
+  # Retrieve the environment
+  env = model.get_env()
+  # Train the agent
+  model.learn(total_timesteps=100000)
+  # Save the agent
+  model.save("ppo2-cartpole")
+
+  del model
+  # the policy_kwargs are automatically loaded
+  model = PPO2.load("ppo2-cartpole")
+
+
+You can also easily define a custom architecture for the policy (or value) network:
```

Comment:
Now that we have policy_kwargs, the following code block seems rather confusing. Why should I create my custom policy class that just changes the parameters to its superclass? To justify custom policy classes we would need a more complex example here.
My first idea was a LSTM network with a MLP stream that skips over the LSTM cells. But then it becomes all very hard to read for an example ...

Reply:
Good point, after writing the example I also wonder if we still need the next example.
To answer your question, I'll take one use case that I have: usually, I have one or two policy architecture (one "big" and one "shallow") and creating a custom policy rather than passing kwargs allow me to name it, so it is quite clear which policy is used.
I should maybe update that line, to say "you can name and define custom policies that way"